wpeverest/user-registration
# Fix/date picker

### All Submissions:






### Changes proposed in this Pull Request:
This PR fixes the issue of date field: Date picker is not showing when there are multiple date fields in a form

### How to test the changes in this Pull Request:

1. Create a form
2. Add multiple date fields
3. Go to the frontend and see if everything is working properly

### Types of changes:




### Other information:




### Changelog entry

> Fix - Date picker not showing when there are multiple Date fields.
